src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestElement.cs

```diff
@@ -161,6 +162,20 @@ public TestCategoryItemCollection TestCategories
             }
         }
 
+        /// <summary>
+        /// Gets or sets the work items.
+        /// </summary>
+        public WorkitemCollection Workitems
```

Comment:
Same here, please make sure you match the case.

Reply:
I updated all casings.

I had to make some more changes in order to keep the custom XML-serialization implementation happy. Unfortunately,  I will not be able to test these changes myself until _Thursday_, so I cannot promise it actually works.

I will keep you updated about the tests, thank you for the review!

Reply:
Yep. I'll verify the changes, and fix it if needed.